pylint/plugins/hass_imports.py

```diff
@@ -398,6 +398,18 @@ class ObsoleteImportMatch:
             constant=re.compile(r"^save_json|find_paths_unserializable_data$"),
         ),
     ],
+    "typing": [
+        ObsoleteImportMatch(
+            reason="should be imported from collections.abc",
+            constant=re.compile(r"^(Async)?Generator$"),
+        ),
+    ],
```

Comment:
That's already handled by the builtin pylint `typing` plugin which we have enabled.

Reply:
Removed